package.json

```diff
@@ -3,11 +3,12 @@
   "version": "1.0.0",
   "description": "A collection of migration recipes for userland code.",
   "scripts": {
-    "test:e2e": "node --no-warnings --experimental-test-coverage --test-reporter=lcov --test-reporter-destination=./coverage.lcov --test-reporter=spec --test-reporter-destination=stdout --import './build/snapshots.ts' --test --test-coverage-include='recipes/**/*' --test-coverage-exclude='**/*.e2e.m(j|t)s' './packages/*/*.e2e.m(j|t)s'",
-    "lint": "biome lint ./",
-    "lint:fix": "biome lint --fix ./",
-    "format": "biome format ./",
     "format:fix": "biome format --fix ./",
+    "format": "biome format ./",
+    "lint:fix": "biome lint --fix ./",
+    "lint": "biome lint ./",
+    "pre-commit": "node --run lint:fix; node --run format:fix; node --run test:types; node --run test:unit; node --run test:e2e",
```

Comment:
why not use `&&` instead of `;` to a have a "strict" chain if one fail rest will not follow

Reply:
I considered that. I did it this way because they're independent, and allowing others to continue at least gets it closer (and perhaps the failures can be fixed manually).